### ISSUE TITLE
"Time Scale": Missing action, misleading naming, inconsistent grouping

### ISSUE THREAD

**Issue by Elairyx7301** [4ian/GDevelop#3248]
## Important note
**This is neither a real bug nor a feature request!**
As the title implies, this issue will mention a missing action, misleading naming and inconsistent grouping of actions/conditions. In lack of a more appropriate category, this issue will not follow the bug issue structure.

"Time Scale": Missing action, misleading naming, inconsistent grouping

### `››` 1. Missing expression for scene time scale

The expression to get the scene time scale is missing in the expression list *[› image 1]*. Since the layer time scale is independent from / isn't changed by the scene time scale, asking for a (e.g. base layer) layer time scale will not work.

**Current workaround:** Set scene time scale to variable and get variable value instead
**Workaround quality:** Poor but working
**Suggested solution:** Add a scene time scale expression

### `››` 2. Inconsistent action groups for time scale

"Change time scale" action is in category or group: "Timers and time"
"Change layer time scale" action is in category or group: "Layers and cameras/Time" (as only entry of the group)
For me it doesn't matter much since I always search for actions (/conditions etc.) but maybe those two actions can be move so that they are in the same group. I see this can be difficult and this issue isn't at a high priority, just throwing that into the room, possibly for discussion.

### `››` 3. Confusing action naming inside expression builder

The method '**LayerTimeScale**(*Layer*)' *[› image 1, expression suggestion]* is called "Time scale" *[› image 2]* in the list inside the expression builder and can easily be confused to be related with the "Change time scale" action *[› image 3]*.

**Suggested solution:** Change name from "Time scale" to "Layer time scale".

---
Related images:

![image](https://user-images.githubusercontent.com/83909121/141658018-3612d6e4-01cf-44e3-be17-33e857d7f868.png)

## Other details
GDevelop 5.0.122-beta1 based on GDevelop.js 4.0.99-0-release
PC / Desktop GDevelop version, Windows
Regular build, no nightly version

**Comment by 4ian** [comment]
> Missing expression for scene time scale

Good catch, this was indeed hidden because of a bug - but turns out you can already use `TimeScale()` expression! I've fixed the visibility of this expression.

> Inconsistent action groups for time scale

Understandable.
For now I let it in the same group, but remove the extra "Time" category that was useless.

> Confusing action naming inside expression builder

You're right, I've changed from "Time scale" to "Layer time scale".

So you'll now have both expressions available with consistent names shown in the editor in the next version :) 

Thanks for reporting this, this will be much better now! Can't believe `TimeScale()` expression has been hidden for years!
